### PR TITLE
Adjust streamlit to PuPr syntax

### PR DESCRIPTION
**Pull request by sfc-gh-turbaszek**
Solves: #442

### Pre-review checklist
   * [ ] I've confirmed that instructions included in README.md are still correct after my changes in the codebase.
   * [ ] I've added new automated tests to verify correctness of my new code.
   * [ ] I've confirmed that my changes are up-to-date with the target branch.
   * [ ] I've described my changes in the release notes.
   * [ ] I've described my changes in the section below.

### Changes description
...

### CODE REVIEW

File: src/snowcli/cli/streamlit/commands.py [hunk 1]
```diff
@@ -29,6 +26,29 @@
 log = logging.getLogger(__name__)
 
 
+StageNameOption: str = typer.Option(
+    "streamlit",
+    "--stage",
+    help="Stage name where are Streamlit files.",
```
Comment (sfc-gh-turbaszek) [inline src/snowcli/cli/streamlit/commands.py:32]:
```suggestion
    help="Stage name where Streamlit files will be uploaded.",
```

File: src/snowcli/cli/streamlit/commands.py [hunk 2]
```diff
@@ -128,51 +114,61 @@ def streamlit_drop(
...
 def streamlit_deploy(
-    name: str = typer.Argument(..., help="Name of streamlit to deploy."),
+    streamlit_name: str = typer.Argument(..., help="Name of Streamlit to deploy."),
     file: Path = typer.Option(
         "streamlit_app.py",
         exists=True,
         readable=True,
         file_okay=True,
         help="Path of the Streamlit app file.",
     ),
```
Comment (sfc-gh-astus) [inline src/snowcli/cli/streamlit/commands.py:124]: `file` don't have flag?

File: src/snowcli/cli/streamlit/manager.py [hunk 3]
```diff
@@ -65,50 +37,57 @@ def share(self, streamlit_name: str, to_role: str) -> SnowflakeCursor:
...
-        if open_in_browser:
-            typer.launch(url)
-        else:
-            return url
+        stage_manager.put(main_file, root_location, 4, True)
+
+        if environment_file and environment_file.exists():
+            stage_manager.put(environment_file, root_location, 4, True)
+
+        if pages_dir and pages_dir.exists():
```
Comment (sfc-gh-astus) [inline src/snowcli/cli/streamlit/manager.py:71]: What if `pages_dir` is not None but does not exists?

File: tests_integration/test_streamlit.py [hunk 4]
```diff
@@ -69,6 +64,7 @@ def test_streamlit_create_and_deploy(
     assert row_from_snowflake_session(expect) == []
 
 
+@pytest.mark.skip("This a PrPr feature of Streamlit, this test can be reused in future")
```
Comment (sfc-gh-astus) [inline tests_integration/test_streamlit.py:67]: Maybe let's just remove it?

File: tests/streamlit/test_commands.py [hunk 5]
```diff
@@ -11,219 +11,202 @@
...
-                create streamlit {STREAMLIT_NAME}
-            
-                MAIN_FILE = 'streamlit_app_launcher.py'
-                QUERY_WAREHOUSE = MockWarehouse;
-                
-                alter streamlit {STREAMLIT_NAME} checkout;
-            """
-        )
+
+def _put_query(source: str, dest: str):
```
Comment (sfc-gh-astus) [inline tests/streamlit/test_commands.py:38]: Maybe `_put_file` or `_put_on_stage`?

File: src/snowcli/cli/streamlit/manager.py (hunk 3 above)
Comment (sfc-gh-turbaszek) [reply src/snowcli/cli/streamlit/manager.py:71]: Good point I will add validation

**Review by sfc-gh-astus — APPROVED** [review]

**Pull request merged**